### ISSUE TITLE
pub get failed (1)

### ISSUE THREAD

**Issue by droplet-js** [Jaguar-dart/jaguar_orm#117]
````
The current Dart SDK version is 2.1.2-dev.0.0.flutter-0a7dcf17eb.





Because jaguar_orm_gen <2.2.2 requires SDK version >=2.0.0-dev.55.0 <2.0.0 and jaguar_orm_gen >=2.2.2 <2.2.18 depends on build ^0.12.0, jaguar_orm_gen <2.2.18 requires build ^0.12.0.

And because jaguar_orm_gen >=2.2.18 depends on analyzer ^0.33.0 and jaguar_serializer_cli 2.2.5 depends on build ^1.0.0, if jaguar_orm_gen any and jaguar_serializer_cli 2.2.5 then analyzer ^0.33.0.

And because jaguar_serializer_cli 2.2.5 depends on analyzer ^0.34.0 and no versions of jaguar_serializer_cli match >2.2.5 <3.0.0, jaguar_orm_gen is incompatible with jaguar_serializer_cli ^2.2.5.
So, because xxx_xxx depends on both jaguar_orm_gen any and jaguar_serializer_cli ^2.2.5, version solving failed.
````

**Comment by tejainece** [comment]
Fixing it.

**Comment by tejainece** [comment]
I have published `jaguar_orm_gen` version `2.2.27`. Let me know if it fixes the issue.

**Comment by droplet-js** [comment]
@tejainece thanks
